docs/API.md

```diff
@@ -554,6 +554,143 @@ try {
 }
 ```
 
+
+<a name="setBucketLifeCycle"></a>
+### setBucketLifeCycle(String bucketName, String lifeCycle)
+`public void setBucketLifeCycle(String bucketName, String lifeCycle)`
+
+Set a life cydle on bucket.
+
+[View Javadoc](http://minio.github.io/minio-java/io/minio/MinioClient.html#setBucketLifeCycle-java.lang.String-java.lang.String-io.minio.BucketLifeCycle-)
+
+__Parameters__
+
+|Param   | Type   | Description  |
+|:--- |:--- |:--- |
+| ``bucketName``  | _String_  | Name of the bucket.  |
+| ``lifeCycle`` | _String_ | Life cycle XML for the bucket. |
+
+| Return Type	  | Exceptions	  |
+|:--- |:--- |
+|  None  | Listed Exceptions: |
+|        |  ``InvalidBucketNameException`` : upon invalid bucket name. |
+|        | ``NoSuchAlgorithmException`` : upon requested algorithm was not found during signature calculation.  |
+|        | ``InsufficientDataException`` : Thrown to indicate that reading given InputStream gets EOFException before reading given length. |
+|        | ``IOException`` : upon connection error.            |
+|        | ``InvalidKeyException`` : upon an invalid access key or secret key.           |
+|        | ``NoResponseException`` : upon no response from server.            |
+|        | ``org.xmlpull.v1.XmlPullParserException`` : upon parsing response XML.            |
+|        | ``ErrorResponseException`` : upon unsuccessful execution.            |
+|        | ``InternalException`` : upon internal library error.        |
+|        | ``InvalidArgumentException`` : upon invalid value is passed to a method.        |
+
+__Example__
+
+
+```java
+try {
+    /* Amazon S3: */
+  MinioClient minioClient = new MinioClient("https://s3.amazonaws.com", "YOUR-ACCESSKEYID",
+                                          "YOUR-SECRETACCESSKEY");
+  String lifeCycle = "<LifecycleConfiguration><Rule><ID>expire-bucket</ID><Prefix></Prefix>"
+                + "<Status>Enabled</Status><Expiration><Days>365</Days></Expiration>"
+                + "</Rule></LifecycleConfiguration>";
+
+
+  minioClient.setBucketLifecycle("lifecycleminiotest", lifeCycle);
+} catch (MinioException e) {
+  System.out.println("Error occurred: " + e);
+}
+```
+
+
```

Comment:
We can remove the extra empty lines in middle

Reply:
removed